<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Discard</class>
 <widget class="QWidget" name="Discard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="discardTable">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>521</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="discardEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>300</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushBtn">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>300</y>
     <width>80</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>чбошод</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
